docs/explanations/about_esm.md

```diff
@@ -61,3 +61,29 @@ You can manage `esm-infra` and `esm-apps` using `pro` on the command line. To
 find out how, read our guide on
 [enabling and disabling these services](../howtoguides/enable_esm_infra.md)
 on your machine.
+
+## Are `esm-infra` and `esm-apps` packages preferred over regular updates?
+
+Yes. The Pro Client will deliver the following configuration files to apt to
+ensure the `esm` updates are preferred when running `apt upgrade`:
+
+- `/etc/apt/preferences.d/ubuntu-pro-esm-infra`
+- `/etc/apt/preferences.d/ubuntu-pro-esm-apps`
+
+Those files are *pinning* the packages coming from `esm` to a higher value than
+the standard defined in apt. This means that the version to be installed by apt
+(in `install` or `upgrade`) or by unattended-upgrades will always be the
+highest esm version available for a given package, even if a higher version is
+present on a non-esm source.
+
+This behavior guarantees that if you have esm-infra or esm-apps enabled, your
+system will always have the esm patches installed for any package available in
+the esm repositories.
+
+Although the files will always be delivered by the package, they will only take
+effect when the packages from those origins are available, i.e. when the
+services are enabled.
```

Comment:
I found this a bit confusing because of the terminology switch. Which files are we referring to here? Do we mean that patches will be downloaded but not installed if the services are disabled?

Reply:
Well it means that, if the services are disabled, there is no harm in keeping those files around

Reply:
and the files are the preferences files... I tried to rewrite it.